freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -348,3 +367,53 @@ def _create_original_to_rebuilt_nvrs_map(self):
                             block_build_nvr['version'] == build_nvr['version']:
                         nvrs_mapping[block_build] = build['nvr']
         return nvrs_mapping
+
+    def _prepare_builds(self, db_event, bundles_by_digest, to_rebuild_digests):
+        """
+        Prepare models.ArtifactBuild instance for every bundle that will be
+        rebuilt
+
+        :param models.Event db_event: database event that will contain builds
+        :param dict bundles_by_digest: mapping of bundle digest to bundle data
+        :param list to_rebuild_digests: digests of bundles to rebuild
+        :return: builds that already in database and ready to be submitted to brew
+        :rtype: list
+        """
+        builds = []
+        csv_mod_url = conf.freshmaker_root_url + "/api/2/pullspec_overrides/{}"
+        for digest in to_rebuild_digests:
+            bundle = bundles_by_digest[digest]
+            # Reset context to db_event for each iteration before
+            # the ArtifactBuild is created.
+            self.set_context(db_event)
+
+            rebuild_reason = RebuildReason.DEPENDENCY.value
+            bundle_name = koji.parse_NVR(bundle["nvr"])["name"]
+
+            build = self.record_build(
+                db_event, bundle_name, ArtifactType.IMAGE,
+                state=ArtifactBuildState.PLANNED.value,
+                original_nvr=bundle["nvr"],
+                rebuild_reason=rebuild_reason)
+
+            # Set context to particular build so logging shows this build
+            # in case of error.
+            self.set_context(build)
+
+            build.transition(ArtifactBuildState.PLANNED.value, "")
+
+            ContainerImage.get_additional_data_from_koji(bundle["nvr"])
+            additional_data = koji
+            build.build_args = json.dumps({
+                "repository": additional_data["repository"],
+                "commit": additional_data["commit"],
+                "target": additional_data["target"],
+                "branch": additional_data["git_branch"],
+                "arches": additional_data["arches"],
+                "operator-csv-modifications-url": csv_mod_url.format(build.id),
```

Comment:
https://github.com/containerbuildsystem/koji-containerbuild/blob/1ddd49d86d1147691f32d71e335af68d4e929563/koji_containerbuild/plugins/builder_containerbuild.py#L663-L666
```suggestion
                "operator_csv_modifications_url": csv_mod_url.format(build.id),
```